File: Aran/Panda/RNAV/PBNAPVBaroVNav/trunk/Forms/AboutForm.cs
using System;
using System.Reflection;
using System.Windows.Forms;

namespace Aran.PANDA.RNAV.PBNAPVBaroVNav
{
	public partial class AboutForm : Form
	{
		public AboutForm()
		{
			InitializeComponent();

			Assembly currentAssem = Assembly.GetExecutingAssembly();

			lbllVersion.Text = "Version :  " + currentAssem.GetName().Version.ToString();
			lblVersionDate.Text = "Build date : " + Functions.RetrieveLinkerTimestamp().ToString("MM.dd.yyyy");

			AssemblyTitleAttribute titleAttrib = (AssemblyTitleAttribute)currentAssem.GetCustomAttribute(typeof(AssemblyTitleAttribute));
			this.Text = "About " + titleAttrib.Title;

			AssemblyDescriptionAttribute descAttrib = (AssemblyDescriptionAttribute)currentAssem.GetCustomAttribute(typeof(AssemblyDescriptionAttribute));
			lblDescription.Text = descAttrib.Description;

			AssemblyCopyrightAttribute copyrightAttrib = (AssemblyCopyrightAttribute)currentAssem.GetCustomAttribute(typeof(AssemblyCopyrightAttribute));
			lblCopyRight.Text = copyrightAttrib.Copyright;
		}

		private void Button1_Click(object sender, System.EventArgs e)
		{
			Close();
		}
	}
}

File: Aran/Panda/SRVectoring/Forms/AboutForm.vb
Imports System.Reflection

Public Class AboutForm
	Public Sub New()
		InitializeComponent()

		Dim currentAssem As Assembly = Assembly.GetExecutingAssembly()

		lbllVersion.Text = "Version :  " + currentAssem.GetName().Version.ToString()
		lblVersionDate.Text = "Build date : " + RetrieveLinkerTimestamp().ToString("MM.dd.yyyy")

		Dim attrib As Object = currentAssem.GetCustomAttribute(GetType(AssemblyTitleAttribute))
		Me.Text = "About " + CType(attrib, AssemblyTitleAttribute).Title

		attrib = currentAssem.GetCustomAttribute(GetType(AssemblyDescriptionAttribute))
		lblDescription.Text = CType(attrib, AssemblyDescriptionAttribute).Description

		attrib = currentAssem.GetCustomAttribute(GetType(AssemblyCopyrightAttribute))
		lblCopyRight.Text = CType(attrib, AssemblyCopyrightAttribute).Copyright
	End Sub

	Private Sub Button1_Click(sender As Object, e As EventArgs) Handles Button1.Click
		Close()
	End Sub
End Class